api/v1/backend.go

```diff
@@ -167,6 +167,18 @@ func (in *S3Spec) RestoreEnvVars() map[string]*corev1.EnvVar {
 		}
 	}
 
+	if in.Region != "" {
+		vars[cfg.AwsDefaultRegion] = &corev1.EnvVar{
+			Name:  "AWS_DEFAULT_REGION",
+			Value: in.Region,
+		}
+	} else {
+		vars[cfg.AwsDefaultRegion] = &corev1.EnvVar{
+			Name:  "AWS_DEFAULT_REGION",
+			Value: cfg.Config.GlobalRestoreS3Region,
+		}
+	}
```

Comment:
Maybe a bit more readable:
```suggestion
	region := cfg.Config.GlobalRestoreS3Region
	if in.Region != "" {
		region = in.Region
	}
	vars[cfg.AwsDefaultRegion] = &corev1.EnvVar{
		Name:  "AWS_DEFAULT_REGION",
		Value: region,
	}
```